File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task71_PERPLEXITY_llama-3-sonar-large-32k-chat.java
public class Main {
    public static void main(String[] args) {
        if (args.length > 0) {
            try {
                int num = Integer.parseInt(args[0]);
                System.out.println("The integer value is: " + num);
            } catch (NumberFormatException e) {
                System.out.println("Invalid input. Please enter a valid integer.");
            }
        } else {
            System.out.println("No input provided.");
        }
    }
}

File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task71_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <cstdlib>

int main(int argc, char *argv[]) {
    if (argc > 1) {
        try {
            int num = std::stoi(argv[1]);
            std::cout << "The integer value is: " << num << std::endl;
        } catch (std::invalid_argument const &e) {
            std::cout << "Invalid input. Please enter a valid integer." << std::endl;
        } catch (std::out_of_range const &e) {
            std::cout << "Invalid input. The number is out of range." << std::endl;
        }
    } else {
        std::cout << "No input provided." << std::endl;
    }
    return 0;
}